planning/behavior_velocity_planner/src/scene_module/intersection/debug.cpp

```diff
@@ -223,7 +223,7 @@ visualization_msgs::msg::MarkerArray IntersectionModule::createVirtualWallMarker
       &wall_marker, now);
   }
 
-  auto id = 0;
+  auto id = planning_utils::bitShift(module_id_);
```

Comment:
@soblin 
memo for confirming my understanding and TODO below
It fixes the problem in the description but  `VirtualWallMarkerCreator` assume id starts from `0`(this is mistake in my opinion)
So deleting virtualwallmarker feature doesn't work properly with this change I thought.
Am I correct?
And I think I need to fix this [part](https://github.com/autowarefoundation/autoware.universe/blob/f944153e492a6e894fd7b02d818c48290d1f5f95/common/motion_utils/src/marker/marker_helper.cpp/#L135-L157) so that deletating feature can deal with the situation where id doesn't start from 0.

Reply:
@kyoichi-sugahara I think there are 2 solutions:
- All behavior velocity planner module start the id from non-zero.
- add prefix to each behavior_velocity_planner modules like as follows:
  - {ns = "intersection1_stop_factor_text", id=0, name="intersection"}, {ns = "intersection1_stop_factor_text", id=1, name="intersection_occlusion"}
  - {ns = "intersection2_stop_factor_text", id=0, name="intersection"}, {ns = "intersection2_stop_factor_text", id=1, name="intersection_occlusion"}

I prefer the 2nd approach.

Reply:
@soblin 
I agree with this part
>I think all of these modules need to start the id from non-zero value to handle this error.
and about your proposed solution, if namespace can be changed for each intersection, then I think we can solve this problem with keeping that id always starts from 0 i guess?

Reply:
I'll merge this for now